libs/blocks/src/lib/footer/footer.social.tsx

```diff
@@ -0,0 +1,18 @@
+import { Text } from '@deriv/quill-design';
+import { SocialIcons } from '.';
+
+export const SocialMediaSection = ({
+  socialButtons,
+}: {
+  socialButtons: SocialIcons[];
+}) => {
+  return (
+    <div className="flex flex-row gap-gap-lg max-lg:justify-center">
```

Comment:
```suggestion
    <div className="flex gap-gap-lg max-lg:justify-center">
```

`flex` direction is `row` by default